src/logion/controllers/adapters/locrequestadapter.ts

```diff
@@ -127,24 +127,22 @@ export class LocRequestAdapter {
 
     async findUserPrivateData(request: LocRequestAggregateRoot): Promise<UserPrivateData> {
         const description = request.getDescription();
-        if (description.requesterIdentityLoc === undefined) {
+        if (description.locType === "Identity" || description.requesterIdentityLoc === undefined) {
```

Comment:
I guess that `description.requesterIdentityLoc === undefined` means "is a Logion Identity"? If that's the case, then the term is redundant with `description.locType === "Identity"`.

Reply:
It's redundant, but the idea was just to avoid the use of `!` when using `requesterIdentityLoc` later on, but indeed this might introduce some misunderstanding. I will change.